src/main/groovy/info/solidsoft/gradle/pitest/PitestTask.groovy

```diff
@@ -299,22 +299,15 @@ class PitestTask extends JavaExec {
     @Override
     void exec() {
         //Workaround for compatibility with Gradle <4.0 due to setArgs(List) and setJvmArgs(List) added in Gradle 4.0
-        args = createListOfAllArgumentsForPit()
+        args = argumentsForPit()
         jvmArgs = ((List<String>)getMainProcessJvmArgs().getOrNull() ?: getJvmArgs())
-        main = "org.pitest.mutationtest.commandline.MutationCoverageReport"
+        main = 'org.pitest.mutationtest.commandline.MutationCoverageReport'
         classpath = getLaunchClasspath()
         super.exec()
     }
 
-    private List<String> createListOfAllArgumentsForPit() {
-        Map<String, String> taskArgumentsMap = createTaskArgumentMap()
-        List<String> argsAsList = createArgumentsListFromMap(taskArgumentsMap)
-        List<String> multiValueArgsAsList = createMultiValueArgsAsList()
-        return argsAsList + multiValueArgsAsList
-    }
```

Comment:
I like to have private methods right after its usage. For most of the 'simple' cases.

Reply:
Will disable PublicMethodsBeforeNonPublicMethods rule